assets/css/base/_base.scss

```diff
@@ -1 +1,18 @@
 // Base styles.
+
```

Comment:
The `$family-sans-serif` variable in `abstracts/_variables.scss` no longer does anything (in the deploy preview, I'm seeing a default serif font rather than Fira Sans). Can you use this variable to apply Fira Sans to the entire page?